app/views/users/show.html.erb

```diff
@@ -1,2 +1,40 @@
 
 <h1>Here show user for given user</h1>
+
+
+  <div class="container">
+      <div class="row">
+          <div class="col-3 border my-3 p-3">
+              
+          </div>
+          <div class="col-9">
+              <div class ="d-flex justify-content-around align-items-baseline border m-3 p-3">
+                <h1><%= @user.name %></h1>
+                <p>Number of posts : <%= @user.posts_counter %></p>
+            </div>
+          </div>
+      </div>
+      <div class="row">
+         <div class="col-12 border ">
+              <h1>Bio</h1>
+            <p><%= @user.bio %></p>
+          </div>
+      </div>
+      <% @posts.each do |post| %>
+      <div class="row mt-3">
+         <div class="col-12 border ">
+              <h1><%= post.title %></h1>
+            <p><%= post.text %></p>
+            <div class ="d-flex flex-row-reverse">
+            <p class="text-end p-2">likes: <%= post.likes_counter %></p>
+            <p class="text-end p-2">comments: <%= post.comments_counter %></p>
+          </div>
+      </div>
+      <% end %>
```

Comment:
- [ ] You should only display the most recent 3 posts of the user instead of all posts. (As seen in wireframe)